runsc/sandbox/network.go

```diff
@@ -264,6 +264,15 @@ func createInterfacesAndRoutesFromNS(conn *urpc.Client, nsPath string, hardwareG
 
 			// Steal IP address from NIC.
 			if err := removeAddress(ifaceLink, addr.String()); err != nil {
+				// If we encounter an error while deleting the ip,
+				// verify the ip is still present on the interface.
+				present, rErr := isAddressOnInterface(iface.Name, addr)
+				if rErr != nil {
```

Comment:
nit: This is a bit more idiomatic:
	if present, err := isAddressOnInterface(iface.Name, addr); err != nil {
		return fmt.Errorf("checking if address %v is on interface %q: %w", addr, iface.Name, err)
	} else if !present {
		continue
	}

Reply:
Thanks, I'll do that.